### PR TITLE
Feature/kotlin tests

### PR DESCRIPTION
**Pull request by idodeclare**
Hello,

Please consider for integration this patch to add Kotlin tests and to fix the exposed bugs.

Thank you.

### CODE REVIEW

**Comment by idodeclare** [comment]
Oops, I realize now this has a dependency on a change made for `kotlin_tests`, so I'll re-raise after that PR is processed.

**Comment by tarzanek** [comment]
please do, I just merged haskell

File: src/org/opensolaris/opengrok/analysis/kotlin/KotlinSymbolTokenizer.lex [hunk 1]
```diff
@@ -23,15 +23,16 @@
  */
 
 /*
- * Gets Java symbols - ignores comments, strings, keywords
+ * Gets Kotlin symbols - ignores comments, strings, keywords
```
Comment (tarzanek) [inline src/org/opensolaris/opengrok/analysis/kotlin/KotlinSymbolTokenizer.lex]: :) thnx

**Comment by tarzanek** [comment]
lgtm actually, so once you're reopen I'd be happy to merge and thanks for the comments detection fix

**Comment by idodeclare** [comment]
> thanks for the comments detection fix

OK you're welcome.

**Pull request closed without merging**